test/grouping.jl

```diff
@@ -291,4 +291,22 @@ end
 	@test @select(g, :a, t = :c).a ≅ d.a
 end
 
+
+@testset "orderby with a grouped data frame" begin
+    df = DataFrame(
+        g = [1, 1, 1, 2, 2],
+        i = 1:5,
+        t = ["a", "b", "c", "c", "e"],
+        y = [:v, :w, :x, :y, :z],
+        c = [:g, :quote, :body, :transform, missing]
+        )
+
+    gd = groupby(df, :g)
+
+    @test @orderby(gd, mean(:i)).i == [1, 2, 3, 4, 5]
+    @test @orderby(df, std(:i) .- :i).i == [5, 4, 3, 2, 1]
+    @test @orderby(gd, :g, -1 .* (:i .- mean(:i))).i == [3, 2, 1, 5, 4]
```

Comment:
Maybe check against `@orderby(@select(gd, ...), ...)`? That will also allow checking other columns.

Also `df` should be `gd` on the second line.

Reply:
Thanks!